demos/config/jenkins.yml

```diff
@@ -4,28 +4,6 @@ jenkins:
     standard:
       excludeClientIPFromCrumb: false
   labelString: flyweight
-  log:
```

Comment:
These logs are overriding the context, hence the build logs are not sent but the logs from this context.